static/js/lang/README.md

```diff
@@ -0,0 +1,29 @@
+# FormatJS i18n instructions
```

Comment:
I was trying to see how close to the proto we can get, but since it is limited to json formatting, we can't get that close. The closest we can get to is what the textproto would look like if the textproto was translated to json.